webapp/src/ts/modules/messages/messages.component.ts

```diff
@@ -54,7 +54,7 @@ export class MessagesComponent implements OnInit, OnDestroy {
     this.subscribeToStore();
     this.tourService.startIfNeeded(this.route.snapshot);
 
-    this.currentLevel = this.sessionService.isOnlineOnly() ? Promise.resolve() : this.getCurrentLineageLevel();
+    this.currentLevel = this.authService.online(true) ? Promise.resolve() : this.getCurrentLineageLevel();
```

Comment:
I first did this change because of another logic that I moved out of here, but since this function prints a message that is good for debugging then decided to leave it there. 
It's essentially the same as `this.sessionService.isOnlineOnly()`